### PR TITLE
Release Notes for v0.10.4

### PR DESCRIPTION
**Pull request by matthewfcarlson**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -35,6 +35,15 @@ NOTE: It is strongly recommended that you use python virtual environments.  Virt
 
 ## Release Version History
 
+### Version 0.10.4
+
+* Features:
+  * If multiple classes are found (ie when searching for a SettingsManager), it will now pick the one that is deepest in hierarchy from the desired class
+  * Updated parsers for DSC, FDF, and DEC
+* Bugs
+  * Fixed bugs #42, 48, 49, and 51
```
Comment (spbrogan) [inline readme.md]: i didn't see 42 in the milestone

Comment (matthewfcarlson) [reply readme.md]: It wasn't in the milestone but we've merged it since the last release so I figured why not add it

**Comment by spbrogan** [comment]
how about this


### Version 0.10.4

* Bugs:
  * #42 - Fix bug causing incorrect decoding in FMP capsule when called twice
  * #41, #58 - Improve Macro resolution in edk2 parsers
   * #49  - Wincert class had bug in write routine
   * #51 - UEFI Status Code did not support error, warning, info bits and did not safely parse the input.

**Comment by matthewfcarlson** [comment]
Looks great, I'll update

File: readme.md (hunk 1 above)
Comment (spbrogan) [reply readme.md]: can we merge in the #53 so that all the class resolution bugs are in

**Pull request merged**